### PR TITLE
Add docs for new products integration

### PR DESCRIPTION
**Pull request by denisramiros**
We've added a new approach for product dual-write integration. We've created new data entities, dual-write maps and plugins.

### CODE REVIEW

File: articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Finance and operations apps | Other Dynamics 365 apps | Description
 [Colors](mapping-reference.md#170) | msdyn\_productcolors
 [Configurations](mapping-reference.md#171) | msdyn\_productconfigurations
 [Default order settings](mapping-reference.md#172) | msdyn_productdefaultordersettings |
+[DV released distinct products](mapping-reference.md#242) | Product |  The **DV released distinct products** table is a simpler version of **CDS released distinct products** table. It contains only variant specific information, and relies on Dataverse plugins to set relevant product master information from **msdyn\_sharedproductdetails** to **Product**. Available in solution Dynamics 365 Supply Chain Extended v2.3.4.294
```
Comment (mqueva) [inline articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:47]: table -> entity (for F&O artefacts)

File: articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md [hunk 2]
```diff
@@ -88,10 +90,25 @@ The creation of product families from Dynamics 365 Sales isn't supported with th
 
 The synchronization of products happens from the finance and operations app to Dataverse. This means that the values of the product table columns can be changed in Dataverse, but when the synchronization is triggered (when a product column is modified in a finance and operations app), this will overwrite the values in Dataverse.
 
+### New approach for integration of products
+Starting from Supply Chain Management version 10.0.39 and Dynamics 365 Supply Chain Extended v2.3.4.294 a new approach is introduced for product synchronization to impvove the overall performance.
```
Comment (mqueva) [inline articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:94]: typo: improve

File: articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md [hunk 3]
```diff
@@ -88,10 +90,25 @@ The creation of product families from Dynamics 365 Sales isn't supported with th
...
+In this approach, the new data entities [DV released distinct products](mapping-reference.md#242) and [DV released products](mapping-reference.md#243) only contain  the information that is synchronazied to Dataverse, which makes the data entities simpler, having fewer joints and data sources. And the dataverse plugins are taking care of updating the relevant product information from **msdyn\_sharedproductdetails** to **Product**.
```
Comment (mqueva) [inline articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:95]: remove one of the two spaces between "contain" and "the"

Comment (mqueva) [inline articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:95]: simpler -> more performant

File: articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md [hunk 4]
```diff
@@ -88,10 +90,25 @@ The creation of product families from Dynamics 365 Sales isn't supported with th
...
+ 
+As it is mentioned in [Sync on-demand with the Supply Chain Management pricing engine](../../../fin-ops/data-entities/pricing-engine.md) Dynamics 365 Sales uses  Supply Chain Management pricing engine for price related calculations, so in this approach the UnitCost and SalesPrice are not being synchronized, which solves many isues with dual-write timeouts.
```
Comment (mqueva) [inline articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:97]: we probably want to drop this part.

Comment (denisramiros) [reply articles/fin-ops-core/dev-itpro/data-entities/dual-write/product-mapping.md:97]: So, not mentioning about UnitCost/SalesPrice at all?

**Comment by johnmichalak** [comment]
Employee changes need to be made in the private repo. Closing this PR and replacing it with this one: https://github.com/MicrosoftDocs/Dynamics-365-Operations/pull/17758/files#diff-065bc413650d43455428aea31faab53cdd5799edad5753d3e0bb6127e7cb13ab

**Pull request closed without merging**